app/controllers/pinned_links_controller.rb

```diff
@@ -0,0 +1,81 @@
+class PinnedLinksController < ApplicationController
+  before_action :verify_moderator
+  before_action :set_pinned_link, only: [:edit, :update]
+
+  def index
+    links = if current_user.is_global_moderator && params[:global] == '2'
+              PinnedLink.unscoped
+            elsif current_user.is_global_moderator && params[:global] == '1'
+              PinnedLink.where(community: nil)
+            else
+              PinnedLink.where(community: @community)
+            end
+    @links = if params[:filter] == 'all'
+               links.all
+             elsif params[:filter] == 'inactive'
+               links.where(active: false).all
+             else
+               links.where(active: true).all
+             end
+    render layout: 'without_sidebar'
+  end
+
+  def new
+    @link = PinnedLink.new
+  end
+
+  def create
+    data = pinned_link_params
+    post = !data[:post_id].present? ? nil : Post.where(data[:post_id]).first
+    community = !data[:community].present? ? nil : Community.where(data[:community]).first
+
+    @link = PinnedLink.create data.merge(post: post, community: community)
+
+    attr = @link.attributes.map { |k, v| "#{k}: #{v}" }.join(' ')
+    AuditLog.moderator_audit(event_type: 'pinned_link_create', related: @link, user: current_user,
+                            comment: "<<PinnedLink #{attr}>>")
```

Comment:
Has Rubocop not complained about this? Missing a space on this line to line it up with `event_type`.

Reply:
Apparently your pillow worked. :)